src/ScottPlot/Plottable/RadialGaugePlot.cs

```diff
@@ -0,0 +1,709 @@
+    /// Internally this data is stored in a single array and is converted to angular paramters,
+    /// through ComputeAngularData(), which are more suitable for drawing purposes and stored in a 2D array.
+    /// </summary>
+    public class RadialGaugePlot : IPlottable
+    {
+        #region Properties & fields
+        /// <summary>
+        /// Data to be plotted.
+        /// It's copied from of the data passed to either the constructor or the <see cref="Update(double[])"/> method.
+        /// </summary>
+        protected double[] DataRaw;
+
+        /// <summary>
+        /// Angular data (first column: initial angle; second column: swept angle) computed from <see cref="DataRaw"/>.
+        /// </summary>
+        protected double[,] DataAngular;
+
+        /// <summary>
+        /// The maximum value for scaling the gauges.
+        /// This value is associated to <see cref="StartingAngleGauges"/> + <see cref="AngleRange"/>.
+        /// </summary>
+        protected double MaxScale
+        {
+            get => _MaxScale;
+            set
+            {
+                _MaxScale = value;
+                ComputeAngularData();
+            }
+        }
+        private double _MaxScale;
+
+        /// <summary>
+        /// The minimum value for scaling the gauges.
+        /// This value is associated to <see cref="StartingAngleGauges"/> + <see cref="AngleRange"/>.
+        /// </summary>
+        protected double MinScale
+        {
+            get => _MinScale;
+            set
+            {
+                _MinScale = value;
+                ComputeAngularData();
+            }
+        }
+        private double _MinScale = 0;
+
+        /// <summary>
+        /// The maximum angular interval that the gauges will consist of.
+        /// It takes values in the range [0°-360°], default value is 360°. Outside this range, unexpected side-effects might happen.
+        /// </summary>
+        public double AngleRange
+        {
+            set
+            {
+                // Modify value to be in the range [0, 360]
+                _AngleRange = value > 360 ? 360 : (value < 0 ? 0 : value);
+                ComputeAngularData();
+            }
+            get => _AngleRange;
+
+        }
+        private double _AngleRange = 360;
+
+        /// <summary>
+        /// Labels for each gauge.
+        /// Length must be equal to the length of data passed to either the constructor or the <see cref="Update(double[])"/> method.
+        /// </summary>
+        public string[] GaugeLabels;
+
+        /// <summary>
+        /// Colors for each gauge. These colors are dimmed according to <see cref="DimPercentage"/> to draw the gauges' background.
+        /// Length must be equal to the length of data passed to either the constructor or the <see cref="Update(double[])"/> method.
+        /// </summary>
+        public Color[] GaugeColors;
+
+        /// <summary>
```

Comment:
I think this should be renamed and the comments changed. This looks like its from `RadarPlot`, but there isn't a web here.

Reply:
That's true. I first used `RadarPlot` as a template for `RadialGauge` thinking they would share most of the code. It turned out it wasn't the case. This is one piede of code I forgot to delete because it was linked to the initialization of these `Pen` instances:
https://github.com/ScottPlot/ScottPlot/blob/d7f90e1346e91c60c85a45e2d2f75b36302bd19a/src/ScottPlot/Plottable/RadialGaugePlot.cs#L433-L434

I'll deleted this property and use a dummy color for the instances' initialization.